src/plugins/data/common/search/tabify/tabify_docs.ts

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { SearchResponse } from 'elasticsearch';
+import { first, each, isPlainObject } from 'lodash';
+import { IndexPattern } from '../../index_patterns/index_patterns';
+import { Datatable, DatatableColumn } from '../../../../expressions/common';
+
+export function flattenHit(
```

Comment:
In general what's the difference in this `flattenHit` implementation vs index patterns, and should we share them as a util?

Also does this one need to be exported at all if we don't share it elsewhere?

Reply:
i think flattenHit on index pattern doesn't make any sense, so I am exporting this and as a followup we should refactor all the places using indexPattern.flattenHit to migrate to this one and we can remove the indexPattern one then.

Reply:
++ Makes sense, I like the idea of getting rid of it from index pattern